.github/workflows/commitlint.yml

```diff
@@ -2,7 +2,8 @@
 
 name: Lint Commit Messages
 
-on: pull_request
+on:
```

Comment:
Similar comment as above for this change.

Reply:
Roll-backed this commit in this pull request